### ISSUE TITLE
Could you demonstrate how to build in Windows in document?

### ISSUE THREAD

**Issue by dlintw** [go-vgo/robotgo#324]
- Robotgo version (or commit ref):
- Go version: go version go1.16.3 windows/amd64
- Gcc version: gcc.exe (tdm64-1) 9.2.0
- Operating system and bit: Windows 10
- Resolution:
- Can you reproduce the bug at [Examples](https://github.com/go-vgo/robotgo/blob/master/examples/main.go):
  - [v] Yes (provide example code)
  - [ ] No
  - [ ] Not relevant
- Provide example code:

```Go
package main

import (
	"github.com/go-vgo/robotgo"
)

func main() {
  robotgo.ScrollMouse(10, "up")
  robotgo.MouseClick("left", true)
  robotgo.MoveMouseSmooth(100, 200, 1.0, 100.0)
}
```
- Log gist:

## Description
C:\Users\dlin\scoop\apps\go\current\pkg\tool\windows_amd64\link.exe: running gcc failed: exit status 1
C:/TDM-GCC-64/bin/../lib/gcc/x86_64-w64-mingw32/9.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: cannot find -lz
collect2.exe: error: ld returned 1 exit status
...

**Comment by vcaesar** [comment]
You can check the gcc environment, and you can download `https://sourceforge.net/projects/mingw-w64/files/` x86_64-posix to try.

**Comment by vcaesar** [comment]
Closed because of duplicates, #100. All code tests and C dependencies use MinGw-64, TDM-GCC has not been tested.
But you can compile `lpng and lzib` yourself, and you can download https://sourceforge.net/projects/mingw-w64/files/ x86_64-posix.
